### PR TITLE
Fix wrong shape after ToTensorV2 in _check_data_post_transform

### PR DESCRIPTION
**Pull request by Dipet**

### CODE REVIEW

**Review by ternaus — APPROVED** [review]

**Comment by ternaus** [comment]
Version bump?

**Review by creafz — APPROVED** [review]

File: albumentations/core/composition.py [hunk 1]
```diff
@@ -199,7 +199,12 @@ def __call__(self, *args, force_apply=False, **data):
         return data
 
     def _check_data_post_transform(self, data):
-        rows, cols = data["image"].shape[:2]
+        img = data["image"]
+        if isinstance(img, np.ndarray):
```
Comment (BloodAxe) [inline albumentations/core/composition.py:203]: Maybe we introduce a helper function like this to get a shape from image/tensor/etc for a better clarity

`def infer_shape(img: Any) -> Tuple[int,int]:
    """
    TODO: Add comment
    """
    if isinstance(img, np.ndarray):
        return img.shape[:2]

    try:
        import torch
        if torch.is_tensor(img):
            return img.shape[-2:]
    except ImportError:
        pass

    raise RuntimeError("Unsupported type")        `

**Comment by Dipet** [comment]
I think to merge #958 after this PR and push version into PYPI

File: albumentations/core/composition.py (hunk 1 above)
Comment (Dipet) [reply albumentations/core/composition.py]: Done

**Review by BloodAxe — APPROVED** [review]

**Pull request merged**